### PR TITLE
Update Jetson ffmpeg patch for Jetpack 5.1.2 compatibility

### PR DESCRIPTION
**Pull request by AndBobsYourUncle**
This PR updates the script that builds the custom ffmpeg version for Jetson devices to use an updated commit that intelligently does not link the `nvbuf_utils` library if it detects the presence of the NvUtils API.

This allows Frigate to run on a Jetson device running Jetpack 5.1.2 and newer that lacks the `nvbuf_utils` shared library.

The commit sha that is referenced is from a fork of the original repo used from @madsciencetist, with a change to the `CMakeLists.txt` file so that it leaves out the linked library.

It can always be switched back to point to @madsciencetist's repository later if this PR is merged in, but it might be a good idea to make the ffmpeg build script here in Frigate use a sha instead of master so that Docker caches are busted when new changes are required.

Here is the PR against jetson-ffmpeg:
https://github.com/madsciencetist/jetson-ffmpeg/pull/3

I have built this locally, and ran it successfully on my Nvidia Jetson AGX Orin device, running Jetpack 5.1.2. Detections were working, and it appeared that everything was using GPU resources, verified by running jtop.

### CODE REVIEW

**Comment by NickM-27** [comment]
Do we know this won't cause issues for users on Jetson 4?

**Comment by AndBobsYourUncle** [comment]
@NickM-27 

> Do we know this won't cause issues for users on Jetson 4?
Great question. I _could_ reflash my Jetson to Jetpack 4, but it would be a very big pain to reinstall everything all over again that I have installed on it.

Might be quicker to get someone else to retest to make sure.

However, just logically following how everything works, this shouldn't affect Jetson 4 at all. Honestly, from looking at it all, it doesn't even seem like Jetpack 4 devices needed the shared library either, since @madsciencetist was installing `jetson_multimedia_api` for Jetpack 4 as well:
https://github.com/blakeblackshear/frigate/blob/dev/docker/tensorrt/build_jetson_ffmpeg.sh#L22

**Comment by AndBobsYourUncle** [comment]
@NickM-27 

Actually, scratch that, I don't think I can use Jetpack 4 on my AGX Orin, because support for that board was added in Jetpack 5.

**Review by NickM-27 — APPROVED** [review]

**Comment by AndBobsYourUncle** [comment]
@NickM-27 

Thank you for the approval! Any next steps for this PR? I'm new to contributions for this repository.

**Comment by blakeblackshear** [comment]
A comment or approval from @madsciencetist would be good here since they are listed as the codeowner for these images.

**Comment by AndBobsYourUncle** [comment]
@blakeblackshear 

Sounds good. It would be great to be able to get this merged in time to make it into v0.13, but I understand waiting for the proper PR approvals.

**Comment by madsciencetist** [comment]
This change looks good to me. I can't test it, but it _should_ work. If it doesn't, Jetpack 4 (Jetson Nano) users will complain about video not working, and ffmpeg logs will complain about a missing library.

**Pull request merged**
